README.md

```diff
@@ -9,49 +9,53 @@
 
 docker-compose run --rm pycourse
 
-##Main topics:
-
 ### What is python
+* Brief python backstory
```

Comment:
maybe it is a good idea to tell that you can see the Zen of Python by typing ```import this``` on python interactive shell :)
```python3
$ ipython
Python 3.6.5 (default, Mar 30 2018, 06:41:53)
Type 'copyright', 'credits' or 'license' for more information
IPython 6.4.0 -- An enhanced Interactive Python. Type '?' for help.

In [1]: import this
The Zen of Python, by Tim Peters

Beautiful is better than ugly.
Explicit is better than implicit.
Simple is better than complex.
Complex is better than complicated.
... 
```

Reply:
I have completely forgotten about this! I think we must mention this but keeping it on the list makes it more visible ;)